samples/artifacts/DataSummarization/NamedAggregation/NamedAggregation.siddhi

```diff
@@ -0,0 +1,96 @@
+@App:name("NamedAggregation")
+@App:description("Demonstrates how to calculate aggregates for various time granularity")
+
+/*
+Purpose:
+    This application demonstrates how to calculate aggregates for various time granularity.In this example the named aggregation gives the aggregation of events such as average of price and sum of price and top maximum price items with in last 5hours.
+
+Prerequisites:
+    1) Ensure that MySQL is installed on your machine.
+    2) Add the MySQL JDBC driver into {Siddhi_HOME}/lib as follows:
+           a) Download the JDBC driver from: https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.45.tar.gz
+           b) Unzip the archive.
+           c) Copy mysql-connector-java-5.1.45-bin.jar to {Siddhi_Home}/lib directory.
+    3) Create a database named 'production' in MySQL. This database is referred to with the 'jdbc:mysql://localhost:3306/production' url.
+    4)In the store configuration of this application, replace 'username' and 'password' values with your MySQL credentials.
+    5) Save this sample
+       
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'
+    2) If the Siddhi application starts successfully, the following messages would be shown on the console
+        * NamedAggregation.siddhi - Started Successfully!
+        
+Testing the Sample:
+     1) The '{SiddhiDistribution_Home}/samples/artifacts/DataSummarization/NamedAggregation/example.csv' file is used to provide the sample data for TradeStream. To simulate the events in this CSV file, execute the following steps,
+        a) Click on 'Event Simulator' (double arrows on left tab)
+        b) Click 'Feed Simulation' -> 'Create'
+        c) Give a name, description and time interval
+        d) Select 'CSV file' as the 'Simulation Source'
+        e) Click on 'Add Simulation Source'
+        f) Select 'NamedAggregation' as 'Siddhi App Name'
+        g) Select 'TradeStream' as 'StreamName'
+        h) Click on 'Upload' button under 'CSV file' section and upload the '{SiddhiDistribution_Home}/samples/artifacts/DataSummarization/NamedAggregation/example.csv' file
+        i) Give ',' as the delimiter
+        j) Save the configuration.
+        
+    2) Click on the start button (Arrow symbol) next to the newly created simulator, and wait few seconds to complete the  events sending from the '{SiddhiDistribution_Home}/samples/artifacts/DataSummarization/NamedAggregation/example.csv' file to TradeStream.
+    
+    3)  Click on 'Event Simulator' (double arrows on left tab)
+           a) Click 'Single Simulation' (this will be already selected)
+           b) Select 'NamedAggregation' as Siddhi App Name
+           c) Select 'DashBoardStream' as StreamName
+           d) Provide symbol values
+              - symbol: WSO10
+           e) Send event
```

Comment:
Give a small description on what events are expected, i.e per hour granularity level summarizations for each symbol

Reply:
9ccf6a40df54b295ecf702859004e06cc81436b3 This commit fixed the above correction